src/components/ProjectCreateForm/ProjectCreateForm.module.css

```diff
@@ -0,0 +1,14 @@
+.FormActions {
```

Comment:
This is the duplicate

Reply:
i create `FormActions` component inside `goals` project and create issue to put this component to `bricks`

issue: https://github.com/taskany-inc/issues/issues/2329